packages/lib/auth.ts

```diff
@@ -54,7 +54,14 @@ export function isPasswordValid(password: string, breakdown?: boolean, strict?:
       if (password[i] === password[i].toLowerCase()) low = true;
     }
   }
-  return !!breakdown ? { caplow: cap && low, num, min, admin_min } : cap && low && num && min;
+
+  if (!breakdown) return cap && low && num && min && (strict ? admin_min : true);
```

Comment:
I DID also add the admin comparison to the non breakdown return. I guess that was also missing and should be in there, right? Otherwise the `!breakdown` version might show true incorrectly.